lib/libzutil/os/linux/zutil_import_os.c

```diff
@@ -74,7 +74,8 @@
 #endif
 #include <blkid/blkid.h>
 
-#define	DEFAULT_IMPORT_PATH_SIZE	9
+#define	nitems(arr) (sizeof (arr) / sizeof (*arr))
```

Comment:
nit: you should be able to use the `ARRAY_SIZE` macro which is already defined for this.  Though if it's not available here for some reason I'm fine with adding `nitems()` here.

Reply:
I used it for parity with FreeBSD, which supplies it in sys/param.h (I think?), but seems like `ARRAY_SIZE` is commoner, yeah; updated